### PR TITLE
added unmagic.1.0.3

### PR DESCRIPTION
**Pull request by camlspotter**

### CODE REVIEW

**Comment by camlspotter** [comment]
It requires updated spotlib for 4.06.0

File: packages/unmagic/unmagic.1.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+]
+remove: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-uninstall" ]
+]
+depends: [
+  "ocamlfind" { build }
+  "omake" { build & < "0.10" }
+  "spotlib"
+  "typerep"
```
Comment (jpdeplaix) [inline packages/unmagic/unmagic.1.0.3/opam]: This dependency needs a version constraint ```>= "v0.10.0"```

File: packages/unmagic/unmagic.1.0.3/opam [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "ppx_typerep_conv" { build }
+  "ppx_deriving"
+]
+available: [
+  ocaml-version >= "4.06.0"
```
Comment (jpdeplaix) [inline packages/unmagic/unmagic.1.0.3/opam]: I'm curious. Where does this restriction come from ?

Comment (camlspotter) [reply packages/unmagic/unmagic.1.0.3/opam]: RIght, it does not require the ocaml-version constraint.  Thanks!

**Comment by rgrinberg** [comment]
@jpdeplaix ping?

**Comment by jpdeplaix** [comment]
@rgrinberg @camlspotter https://github.com/ocaml/opam-repository/pull/11102#issuecomment-355743858

File: packages/unmagic/unmagic.1.0.3/opam (hunk 1 above)
Comment (jpdeplaix) [reply packages/unmagic/unmagic.1.0.3/opam]: @camlspotter there is still this constraint to add and I'm willing to merge this asap.

**Comment by jpdeplaix** [comment]
poke @camlspotter

**Comment by camlspotter** [comment]
Added a version constraing: `  "typerep" {>= "v0.10.0"}`

**Comment by jpdeplaix** [comment]
Thanks.
Ok I give up. If you want other people to use it with older versions of OCaml, feel free to open an other PR.
Merging.

**Pull request merged**